src/main/java/com/softserve/academy/spaced/repetition/service/RegistrationService.java

```diff
@@ -56,6 +70,12 @@ public User createNewUser(User user) {
         user.getAccount().setAuthorities(Collections.singleton(authority));
         user.getAccount().setLearningRegime(LearningRegime.CARDS_POSTPONING_USING_SPACED_REPETITION);
         user.getAccount().setCardsNumber(CARDS_NUMBER);
+        rememberingLevelRepository.save(new RememberingLevel(1, "Teapot", 1, user.getAccount()));
```

Comment:
This is a huge copy-paste.

Reply:
I know but I have no idea have to fix that. Could you give me a piece of advice?

Reply:
We can discuss on Thursday.

Reply:
Okay, great.

Reply:
a lot of code is duplicated in 3 files

Reply:
And will it be good to create some method, for examle, in UserService, a public static method, which will initialize duplicated fields? Or maybe in RegistrationService?